oceanstream/settings/config.py

```diff
@@ -12,6 +12,6 @@ def load_config(user_config_path=None):
         with open(user_config_path, "r") as file:
             user_config = json.load(file)
         for key in user_config:
-            config[key].update(user_config[key])
```

Comment:
This did not work as expected. config[key] returns the value for that key. update does not work on random data types only on dict. So this was treating the value stored in the dictionary config at key as a dictionary.

update docs https://www.programiz.com/python-programming/methods/dictionary/update